tests/prism/test_cidc_data_model.py

```diff
@@ -0,0 +1,122 @@
+import pytest
+from copy import deepcopy
+from deepdiff import DeepDiff, grep
+
+from cidc_schemas.json_validation import load_and_validate_schema
+from cidc_schemas.prism import (
+    prismify,
+    merge_clinical_trial_metadata,
+    merge_artifact,
+    merge_artifact_extra_metadata,
+)
+
+from .cidc_test_data import list_test_data, PrismTestData
+
+
+@pytest.fixture(params=list_test_data())
+def prism_test(request):
+    return request.param
+
+
+def assert_metadata_matches(received: dict, expected: dict, upload_entries: list):
+    """Check that the `received` metadata dict matches the `expected` metadata dict."""
+    # Take the difference of the received patch and the expected patch.
+    # We expect these patchs to differ by upload placeholder UUID only,
+    # so the number of differences should equal the number of expected
+    # upload entries.
+    diff = DeepDiff(received, expected)
+    if "values_changed" in diff:
```

Comment:
nit: shouldn't this be more like
```suggestion
    if upload_entries:
```